action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Run tflint with reviewdog'
+description: '🐶 Run tflint with reviewdog on pull requests to enforce best practices'
+author: 'vlaaaaaaad (reviewdog)'
+
+inputs:
```

Comment:
Can you add `reporter` to change reviewdog reporter between github-pr-review and github-pr-check for the consistency? 
I added `reporter` flag to actions I created recently (e.g.
https://github.com/reviewdog/action-shellcheck) and I plan to add the same flag to existing actions which don't have the flag yet.

Reply:
Actually, I think it's optional for the initial version, You can keep it as is in this PR if you want.

Reply:
I like the idea, adding it.

Reply:
Added, tested.